ui/src/components/Common/Demo/Demo.vue

```diff
@@ -89,4 +96,16 @@ const cities = ref( [
   { name: 'Istanbul', code: 'IST' },
   { name: 'Paris', code: 'PRS' }
 ]);
+
+const categories = ref([
+            {name: 'Accounting', key: 'A'}, 
+            {name: 'Marketing', key: 'M'}, 
+            {name: 'Production', key: 'P'}, 
+            {name: 'Research', key: 'R'}
+        ]);
+const selectedValue = (val : string) => {
+  console.log('selected value :',val);
```

Comment:
Needs formatting according to the .prettierrc file

Reply:
Formatting is changed.